src/transformers/models/swin/convert_swin_simmim_to_pytorch.py

```diff
@@ -162,7 +162,6 @@ def convert_swin_checkpoint(model_name, checkpoint_path, pytorch_dump_folder_pat
         "--model_name",
         default="swin-base-simmim-window6-192",
         type=str,
-        choices=["swin-base-simmim-window6-192", "swin-large-simmim-window12-192"],
```

Comment:
Any reason this gets removed?

Reply:
Removed accidentally, I'm adding it back